contracts/libraries/VaultLifecycleSTETH.sol

```diff
@@ -322,7 +327,8 @@ library VaultLifecycleSTETH {
                 withdrawAmount.sub(yieldTokenBalance)
             );
 
-        transferAsset(payable(recipient), underlyingTokensToWithdraw);
+        IWETH(weth).deposit{value: underlyingTokensToWithdraw}();
+        IERC20(weth).safeTransfer(recipient, underlyingTokensToWithdraw);
```

Comment:
What is the rationale behind this change?

Reply:
if we use curve, the burner just does swapExactTokensForTokens. so figure it will be easier later down the line for RBN buyback to just have all ERC-20 tokens